city_scrapers/spiders/cook_justice_advisory.py

```diff
@@ -0,0 +1,178 @@
+        """
+        address = response.xpath(
+            '//div[@class="field event-location"]/descendant::*/text()'
+        ).extract()
+        for word in ["Location:", ", ", " "]:
+            address.remove(word)
+        address = " ".join(address)
+        if "Microsoft Teams" in address:
+            return {
+                "address": "",
+                "name": "",
+            }
+        else:
+            return {
+                "address": address,
+                "name": "",
+            }
+
+    def _parse_all_day(self, response):
+        """
+        Parse or generate all-day status. Defaults to false.
+        """
+        date = response.xpath(
+            '//span[@class="date-display-single"]/descendant-or-self::*/text()'
+        ).extract()
+        date = "".join(date).upper()
+        return "ALL DAY" in date
+
+    def _parse_title(self, response):
+        """Parse or generate event"""
+        title = "".join(response.css("h1::text").extract())
+        if "JAC Council Meeting" in title:
+            return "JAC Council Meeting"
+        else:
+            return title
+
+    def _parse_description(self, response):
+        """Parse or generate event description."""
+        category_field = response.xpath(
+            "//div[contains(., 'Category:') and contains(@class, 'field-label')]"
+        )
+        field_items = category_field.xpath(
+            "./following::div[contains(@class, 'field-items')]"
+        )
+        return " ".join(
+            field_items.xpath(".//p/text()").extract()
+            + field_items.xpath(".//strong/text()").extract()
+        ).strip()
+
+    def _parse_start(self, response):
+        """Parse start date and time"""
+        start = response.xpath(
+            '//span[@class="date-display-single"]/descendant-or-self::*/text()'
+        ).extract()
+        start = "".join(start).upper()
+        start = start.split(" TO ")[0].strip()
+        start = start.replace("(ALL DAY)", "12:00AM")
+
+        return datetime.strptime(start, "%B %d, %Y %I:%M%p")
+
+    def _parse_end(self, response):
+        """Parse end date and time"""
+        date = response.xpath(
+            '//span[@class="date-display-single"]/descendant-or-self::*/text()'
+        ).extract()
+        date = "".join(date).upper()
+        date.replace("(ALL DAY)", "TO 11:59PM")
+        start_end = date.split(" TO ")
+
+        if len(start_end) < 2:
+            return
+
+        end_time = start_end[1]
+        date = start_end[0][: start_end[0].rindex(" ")]
+        return datetime.strptime("{} {}".format(date, end_time), "%B %d, %Y %I:%M%p")
+
+    def _get_agenda(self):
+        return CookJusticeAdvisorySpider.agenda_map
+
+    def _parse_links(self, response):
+        """Parse links"""
+        agenda_map = self._get_agenda()
+        files = response.css("span.file a")
+        files = files[2:]
+        for f in files:
+            link = f.xpath("./@href").extract_first()
```

Comment:
It looks like there are some files that don't follow this pattern like the minutes for July 2017: https://www.cookcountyil.gov/sites/default/files/jac_council_minutes_finaldraft.pdf

It might be easier to check for a month in either "%b" or "%B" format and then the year in the text of the link rather than the `href` attribute. That way, it can be parsed to a datetime object and then converted back to a string of "%m-%Y" for easier matching in `_parse_event`